account-kit/smart-contracts/src/ma-v2/modules/native-token-limit-module/module.ts

```diff
@@ -0,0 +1,21 @@
+import { encodeAbiParameters, type Address, type Hex } from "viem";
+
+import { nativeTokenLimitModuleAbi } from "./abis/nativeTokenLimitModuleAbi.js";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'nativeTokenLimitModuleAbi' is defined but never used.
<!-- __reviewdog__:ChA4YWQ4MTliZDk4Y2JlYmMyEgZyZGpzb24= -->